File: app/src/main/java/com/ericho/coupleshare/frag/BaseFrag.java
package com.ericho.coupleshare.frag;

import android.support.v4.app.Fragment;

import butterknife.Unbinder;

/**
 * Created by EricH on 10/5/2017.
 */

public abstract class BaseFrag extends Fragment {
    Unbinder unbinder;


    @Override
    public void onDestroyView() {
        if (unbinder != null) {
            unbinder.unbind();
        }
        super.onDestroyView();
    }
}

File: app/src/main/java/com/ericho/coupleshare/frag/BaseFrag.kt
package com.ericho.coupleshare.frag

import android.support.v4.app.Fragment


/**
 * Created by steve_000 on 8/7/2017.
 * for project CoupleShare
 * package name com.ericho.coupleshare.frag
 */
open class BaseFrag:Fragment() {


    override fun onDestroy() {
        super.onDestroy()
//        val refWatcher = App.getRefWatcher(activity)
//        refWatcher.watch(this)
    }
}